### PR TITLE
Add the single-gain feature

### LINKED ISSUE #244: Add the single-gain feature
Add the single-gain feature for rxgain and txgain.

### PR DESCRIPTION
**Pull request by mindy-min-zhang**
See #244

Closes #244

### CODE REVIEW

File: src/agora/radio_lib.cpp [hunk 1]
```diff
@@ -247,34 +247,48 @@ void RadioConfig::configureBSRadio(RadioConfigContext* context)
...
-                if (_cfg->freq > 3e9)
-                    baStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -6); //[-18,0]
-                else if (_cfg->freq > 2e9 && _cfg->freq < 3e9)
-                    baStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -18); //[-18,0]
-                else
-                    baStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -12); //[-18,0]
-                baStn[i]->setGain(SOAPY_SDR_RX, ch, "LNA2", 17); //[0,17]
-            }
+        // Unified gains for both lime and frontend
+        if (_cfg->single_gain()) {
```
Comment (rdoosty) [inline src/agora/radio_lib.cpp:251]: single gain is only valid for Iris case. The 'if (!kUseUHD)` should be outside of this `if`.

File: src/client/client_radio.cpp [hunk 2]
```diff
@@ -182,34 +182,48 @@ void ClientRadioConfig::initClientRadio(ClientRadioConfigContext* in_context)
...
-                if (cfg->freq > 3e9)
-                    clStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -6); //[-18,0]
-                else if (cfg->freq > 2e9 && cfg->freq < 3e9)
-                    clStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -18); //[-18,0]
-                else
-                    clStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -12); //[-18,0]
-                clStn[i]->setGain(SOAPY_SDR_RX, ch, "LNA2", 17); //[0,17]
-            }
+        // Unified gains for both lime and frontend
+        if (_cfg->single_gain()) {
```
Comment (rdoosty) [inline src/client/client_radio.cpp:186]: Same comment as above.

File: src/agora/radio_lib.cpp (hunk 1 above)
Comment (mindy-min-zhang) [reply src/agora/radio_lib.cpp:251]: @tingjunchen Can you please review my changes? Should I change as Rahman suggested?

Comment (mindy-min-zhang) [reply src/agora/radio_lib.cpp]: @rdoosty I have changed according to your comments.

File: src/client/client_radio.cpp (hunk 2 above)
Comment (mindy-min-zhang) [reply src/client/client_radio.cpp]: @rdoosty I have changed according to your comments.

**Review by rdoosty — APPROVED** [review]

**Pull request merged**